### PR TITLE
Update cpu-manager doc to match implementation.

### PR DESCRIPTION
**Pull request by ConnorDoyle**
- Describe new `--cpu-manager-reconcile-period` flag.
- Describe node allocatable requirements for static policy.
- Remove references to CPU pressure and node condition.
- Update interface sketches.
- Update block diagram image.
- Expand static policy description.

### CODE REVIEW

**Comment by sjenning** [comment]
More references to `systemreserved.cpu` and `kubereserved.cpu`.  I guess we refer to this concept enough to need a shorthand for it.  We just might want to explicitly call out a definition.

Other than that, lgtm

File: contributors/design-proposals/cpu-manager.md [hunk 1]
```diff
@@ -182,6 +188,10 @@ node. Once allocated at pod admission time, an exclusive CPU remains
 assigned to a single container for the lifetime of the pod (until it
 becomes terminal.)
 
+The Kubelet requires `kubereserved.cpu` plus `systemreserved.cpu` to be
```
Comment (balajismaniam) [inline contributors/design-proposals/cpu-manager.md]: s/requires `kubereserved.cpu` plus `systemreserved.cpu`/requires the sum of `kubereserved.cpu` and `systemreserved.cpu`

File: contributors/design-proposals/cpu-manager.md [hunk 2]
```diff
@@ -198,6 +208,35 @@ allocated or deallocated.)
...
+
+- **ASSIGNABLE:** Equal to `SHARED - RESERVED`. Exclusive CPUs are allocated
+  from this pool.
+
+- **EXCLUSIVE ALLOCATIONS:** CPU sets assigned exclusively to one container.
+  These are stored as explicit assignments in the state.
+
+When an exclusive allocation is made, the static policy also updates the
+default cpuset in the state abstraction. The CPU manager's periodic
+reconcile loop takes care of rewriting the cpuset in cgroupfs for any
```
Comment (balajismaniam) [inline contributors/design-proposals/cpu-manager.md]: Nit: s/rewiriting/updating

File: contributors/design-proposals/cpu-manager.md [hunk 3]
```diff
@@ -257,23 +296,21 @@ func (p *staticPolicy) UnregisterContainer(s State, containerID string) error {
...
-       NoExecute that prevents BestEffort and Burstable QoS class pods from
-       running on the node. BestEffort and Burstable QoS class pods are
-       evicted from the node.
-
-1. _The shared pool becomes nonempty._
-    1. The CPU manager removes the node condition with effect NoSchedule,
-       NoExecute for BestEffort and Burstable QoS class pods.
+    1. This cannot happen. The size of the shared pool is greater than
+       the number of exclusively allocatable CPUs. The Kubelet requires
+       `kubereserved.cpu` plus `systemreserved.cpu` to be greater than zero
```
Comment (balajismaniam) [inline contributors/design-proposals/cpu-manager.md]: s/requires `kubereserved.cpu` plus `systemreserved.cpu`/requires the sum of `kubereserved.cpu` and `systemreserved.cpu`

**Comment by ConnorDoyle** [comment]
@sjenning and @balajismaniam -- updated, PTAL. Rewrote the kubereserved shorthand.

**Comment by balajismaniam** [comment]
lgtm.

**Pull request merged**